publication-commons/src/main/resources/features/publish_publication.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Publish a Publication
+
+  Scenario: The User publishes a Publication
+    Given the User wants to publish a Publication
```

Comment:
Valid credentials and publishing permission?

Reply:
Who has permissions, the owner and the curator?

Reply:
A question for the PO…